### PR TITLE
Add quickstart for running core tests (not just dev environment)

### PR DESCRIPTION
**Pull request by jdar**
Chris Kosic suggested I send a PR. "we would be happy if you COULD fork/and SEND a PR on this"

Here are some changes that I made, that you may or may not agree with:

I removed reference to 'lerna' since it is no longer required to be a global dependency; it is wrapped by yarn now.

Created a separate page with concise instructions for unit tests. This is very different from setting up a devenv! A separate page with more concise instructions should be less confusing for new devs who are just getting their toes wet.

On this page, I do suggest emailing the GSOD team, since new dev onboarding is a critical issue, and because there seems to be a significant delay for joining the slack. Just a suggestion.

## What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactor
- [ ] Performance
- [ ] Tests
- [ ] Build
- [x ] Documentation
- [ ] Code style update
- [ ] Continuous Integration
- [ ] Other, please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x ] No

## Does this PR release a new version?

- [ ] Yes
- [x ] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [N/A ] It's submitted to the `develop` branch, _not_ the `master` branch
- [N/A ] All tests are passing
- [N/A ] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

**Comment by ghost** [comment]
**Thanks for submitting this pull request!** A maintainer will review this in the next few days and explicitly select labels so you know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

**Comment by ghost** [comment]
Your pull request doesn't follow our [contribution guidelines](https://docs.ark.io/guidebook/contribution-guidelines/). Please review and correct it.

**Comment by jdar** [comment]
> Your pull request doesn't follow our [contribution guidelines](https://docs.ark.io/guidebook/contribution-guidelines/). Please review and correct it.

@ArkEcosystemBot  In what way?

**Comment by jdar** [comment]
If this pull request is accepted, I believe this will address #368

File: docs/guidebook/developer/run-tests-for-first-time.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+If you are a developer, the following steps will help you to validate your installation.
+
+## Run Tests for the First Time (Core)
+
+* Ensure primary dependencies installed:
+  * Ensure `node -v` returns a version higher than or equal to the [minimum recommended version](https://github.com/ArkEcosystem/core/blob/master/.nvmrc)
+  * Ensure yarn is installed. If yarn is not found, you may need to close, exit, and re-open your terminal.
+* checkout git repository for core, located here: [https://github.com/ArkEcosystem/core](https://github.com/ArkEcosystem/core)
+* `cd core`
+* `yarn install`
```
Comment (faustbrian) [inline docs/guidebook/developer/run-tests-for-first-time.md]: `yarn install` is not needed, `yarn setup` will perform that step already.

**Review by faustbrian — APPROVED** [review]

**Comment by ghost** [comment]
A contributor has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Comment by ghost** [comment]
Your pull request has been merged and marked as tier 3. _It will earn you $25 USD._

**Comment by boldninja** [comment]
@jdar Can you provide me with your ARK address for GitHub bounty? You can also contact me on ARK's Slack - https://ark.io/slack (username : Rok [ARK Team]) and direct message me your address there.

**Pull request merged**